src/include/txn_inline.h

```diff
@@ -1930,9 +1930,9 @@ __txn_modify_block(
         }
 
         WT_STAT_CONN_DSRC_INCR(session, txn_update_conflict);
-        ret = __wt_txn_rollback_required(session, WT_TXN_ROLLBACK_REASON_CONFLICT);
+        ret = WT_ROLLBACK;
         __wt_session_set_last_error(
-          session, ret, WT_WRITE_CONFLICT, "Write conflict between concurrent operations");
+          session, WT_ROLLBACK, WT_WRITE_CONFLICT, WT_TXN_ROLLBACK_REASON_CONFLICT);
```

Comment:
```suggestion
          session, ret, WT_WRITE_CONFLICT, WT_TXN_ROLLBACK_REASON_CONFLICT);
```
For consistency with https://github.com/wiredtiger/wiredtiger/pull/11519/files#diff-fba173dbad7997f847117319e1939715f1969181ab2937508a2c9303b201f017R2944